pkgs/games/teeworlds/default.nix

```
@@ -93,7 +93,18 @@ stdenv.mkDerivation rec {
     '';
 
     homepage = "https://teeworlds.com/";
-    license = "BSD-style, see `license.txt'";
+    license = with lib.licenses; [
+      # See https://github.com/teeworlds/teeworlds/blob/master/license.txt
+      zlib # Main license
```

```suggestion
      lib.licenses.zlib # Main license
```

I guess `zlib` evaluates to `pkgs.zlib`, unfortunately.

---

Wow, good catch!